src/apis/DataDisplayApi.ts

```diff
@@ -0,0 +1,19 @@
+
+import { SearchResponse } from '../types/SearchApiTypes';
```

Comment:
We don't have file `../types/SearchApiTypes`. Please define `SearchResponse` locally